From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Wed, 6 Oct 2021 18:41:11 -0700
Subject: [PATCH 12/24] Revert "Do not re-inflate QS and SB when
 CONFIG_UI_MODE"

This reverts commit 8a40ff855b86bc86e23367017002289920855a4e.

This is a prerequisite for making the QS panel follow the light system
mode setting.

Change-Id: Ibad399ece587505559cc73febdda2f2d8558e94d
Signed-off-by: Saalim Quadri <danascape@gmail.com>
(cherry picked from commit 756bd76d33cb6bfcf8852b12953feb58a2536724)
---
 .../com/android/systemui/fragments/FragmentHostManager.java    | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/fragments/FragmentHostManager.java b/packages/SystemUI/src/com/android/systemui/fragments/FragmentHostManager.java
index 643f3bb917d1..822ed088f93f 100644
--- a/packages/SystemUI/src/com/android/systemui/fragments/FragmentHostManager.java
+++ b/packages/SystemUI/src/com/android/systemui/fragments/FragmentHostManager.java
@@ -62,7 +62,8 @@ public class FragmentHostManager {
     private final LeakDetector mLeakDetector;
     private final InterestingConfigChanges mConfigChanges = new InterestingConfigChanges(
             ActivityInfo.CONFIG_FONT_SCALE | ActivityInfo.CONFIG_LOCALE
-                    | ActivityInfo.CONFIG_ASSETS_PATHS);
+                    | ActivityInfo.CONFIG_ASSETS_PATHS
+                    | ActivityInfo.CONFIG_UI_MODE);
     private final FragmentService mManager;
     private final ExtensionFragmentManager mPlugins = new ExtensionFragmentManager();
 
